frontend/src/scenes/experimentation/FeatureFlags.js

```diff
@@ -23,10 +23,12 @@ function _FeatureFlags() {
         {
             title: 'Name',
             dataIndex: 'name',
+            sorter: (a, b) => ('' + a.name).localeCompare(b.name),
```

Comment:
Q: Why the '' + a.name?

Reply:
i just copied this from other instances, it's in case the first string isn't set. But yes we don't actually require it here